CMakeLists.txt

```diff
@@ -15,6 +15,14 @@ source_group(TREE ${CMAKE_CURRENT_LIST_DIR} FILES ${SOURCES})
 add_library (XbSymbolDatabase ${SOURCES})
 target_compile_definitions(XbSymbolDatabase PRIVATE _CRT_SECURE_NO_WARNINGS)
 
+option(BUILD_TOOL "Build XbSymbolDatabase CLI tool" OFF)
+if(BUILD_TOOL)
```

Comment:
Keep in mind, I will move this changes into projects folder.

The goal for each project, such as library, cli tool, and unit test cmake files go into **projects** folder. Plus source files will likely go into their own folder due to multiple projects bundle in one repository. i.e.
```
src/cli/...
src/lib/...
src/unittest/...
```
or without src part.